model_fetch/lib/src/model_by_filter/model_by_filter_bloc.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:meta/meta.dart';
+import 'package:model_interfaces/model_interfaces.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../load_status.dart';
+
+abstract class ModelByFilterBloc<I, T extends WithId<String>> {
+  final I query;
```

Comment:
Delete because this class is not aware of queries.
Also `I` is the type for IDs of objects.